ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -642,7 +643,9 @@ export const AdminInstitutionEdit = fp.flow(withNavigation, withRouter)(class ex
       rtTierEmailDomains: {truthiness: true},
       ctTierEmailDomains: {truthiness: true},
     });
-    return <div>
+    return redirectPath
+        ? <RouteRedirect path={redirectPath}/>
```

Comment:
opt: rather than ternary in these cases, I might slightly prefer an early return; it's mostly unrelated to the mainline rendering:


```
if (redirectPath) {
  return <RouteRedirect .../ >
}
return <div> ...
```